### ISSUE TITLE
Bug: incompatible lastCommitDate format on Firefox

### ISSUE THREAD

**Issue by Delapouite** [Meaningful-Code/meaningfulcode-frontend#7]
Hello.

Here's a excerpt of a project object returned by the API:

```
{
  "url": "https://github.com/covid19india/covid19india-react",
  "websiteUrl": "https://www.covid19india.org",

  "lastUpdate": 1607332409,
  "lastCommitTime": "2020-11-22 05:54:08 +0000 UTC",
}
```

The date format of the `lastCommitTime` field is not ISO8601, which results in [Date.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse) always returning `NaN` on Firefox, which in turn displays `Never` on the UI.

What about using a timestamp like in the `lastUpdate` field to avoid any trouble?

Thanks

**Comment by Pixep** [comment]
Thanks Bruno! Great, thanks for reporting it and the details, will definitely have a look to fix it :+1:

**Comment by Pixep** [comment]
Fix in 7d51f2fff81b2174799225851129f2a297370c70, and deployed. Thanks again :).
